lib/id3c/cli/command/etl/manifest.py

```diff
@@ -206,8 +215,13 @@ def upsert_sample(db: DatabaseSession,
         LOG.info(f"Updating existing sample {sample.id}")
         sample = db.fetch_row("""
             update warehouse.sample
+<<<<<<< HEAD
                set identifier = %(identifier)s,
                    collection_identifier = %(collection_identifier)s,
+=======
+               set identifier = coalesce(%(identifier)s, identifier),
+                   collection_identifier = coalesce(%(collection_identifier)s, collection_identifier),
+>>>>>>> 8033726... Make manifest parsing support multiple source types
```

Comment:
Looks like a git merge conflict slipped into this commit

Reply:
GRRR. Fixed. I saw it. But thanks.